src/helm/benchmark/data_overlap/get_group_scenario_specs.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+import argparse
+from typing import List, DefaultDict, Dict
+from collections import defaultdict
+
+from helm.common.general import asdict_without_nones
+from helm.common.hierarchical_logger import hlog
+
+
+from helm.benchmark.presentation.run_entry import read_run_entries
+from helm.benchmark.run import run_entries_to_run_specs
+from helm.benchmark.data_overlap.light_scenario import GroupScenarioSpecs
+
+
+def save_group_scenario_specs_to_jsonl(group_scenario_specs: List[GroupScenarioSpecs], filename: str):
+    with open(filename, "a") as f:
+        for scenario_spec_groups in group_scenario_specs:
+            f.write(json.dumps(asdict_without_nones(scenario_spec_groups), ensure_ascii=False) + "\n")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--run-specs", nargs="+", required=True, help="Specifies what to export")
+    parser.add_argument("--output-data", type=str, required=True, help="The path to the output file")
+    args = parser.parse_args()
+
+    hlog("Loading run_specs")
+    run_entries = read_run_entries(args.run_specs).entries
+    run_specs = run_entries_to_run_specs(
+        run_entries=run_entries,
+        priority=4,
```

Comment:
why is 4 hardcoded here?

Reply:
We do this to use the same run_conf as the main helm repo to filter those that are not relevant/are problematic to data overlap (e.g. lextreme). I've changed to constant instead so at least it's not a magic number